python/tensorflowjs/converters/tf_saved_model_conversion_v2.py

```diff
@@ -263,131 +261,47 @@ def convert_tf_saved_model(saved_model_dir,
       will consist of
       - a file named 'model.json'
       - possibly sharded binary weight files.
-    signature_def: string Tagset of the SignatureDef to load. Defaulted to
-      'serving_default'
+    signature_name: string Tagset of the SignatureDef to load. Defaults to
+      'serving_default'.
+    saved_model_tags: tags of the GraphDef to load. Unless a list is provided,
+      the provided string will be comma separated. Defaults to 'serve'.
     quantization_dtype: An optional numpy dtype to quantize weights to for
       compression. Only np.uint8 and np.uint16 are supported.
     skip_op_check: Bool whether to skip the op check.
     strip_debug_ops: Bool whether to strip debug ops.
   """
-  if signature_def is None:
-    signature_def = 'serving_default'
+  if not isinstance(saved_model_tags, list):
+    saved_model_tags = saved_model_tags.split(',')
 
   if not os.path.exists(output_dir):
     os.makedirs(output_dir)
   output_graph = os.path.join(
       output_dir, common.ARTIFACT_MODEL_JSON_FILE_NAME)
 
-  saved_model_tags = saved_model_tags.split(', ')
-  model = load(saved_model_dir, saved_model_tags)
-  concrete_func = model.signatures[signature_def]
+  model = hub.load(saved_model_dir, saved_model_tags)
```

Comment:
why use hub to load the model instead of TensorFlow saved model loader?

Reply:
Just to avoid having two code paths doing the same thing.

hub.load = [hub handle resolution] + tf.saved_model.load

Reply:
As I mentioned earlier, we have experienced some issues with V1 model converted by TF 2.0 nightly. Once the 1.14 is released, we will be converting 1.0 model using 1.14, and 2.0 model using compat.v2 in 1.14, which will be incompatible with TFHub nightly. Have TFHub 1.0 modules fully tested with TF 2.0? What version of TensorFlow does TFHub nightly depends on? Thanks.